src/Dont/DeSerialize.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dont;
+
+use Dont\Exception\NonDeSerializableObject;
+use Dont\Exception\TypeError;
+
+trait DeSerialize
```

Comment:
We could always create an alias ? But i'm going to agree with @Ocramius for the simple reason I would be annoyed with phpstorm including the wrong thing.